buildSrc/src/main/java/Gecko.kt

```diff
@@ -0,0 +1,11 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+private object GeckoVersions {
+    const val nightly_version = "67.0.20190206095431"
```

Comment:
Possible that I'm confused here, but if we're declaring this version statically, it's not really nightly, is it?  Should this be renamed?

Reply:
Nightly just means a release channel. Dynamic version updates (especially for dependencies) are generally frowned upon

Reply:
Agreed that we should avoid dynamic versioning, but I do find the name confusing.

On a side note, is there a standard for how to style code shared by two languages?  It's weird to see snake casing in a Kotlin file, but I suppose it's only really going to be used in Gradle.